### DISCUSSION TITLE
Custom DNS entries

### DISCUSSION THREAD

**Discussion by timo0** [AdguardTeam/AdGuardHome#2865] [Q&A]
I would like to add custom CNAME or A entries into the DNS server from AdGuardHome.

The target of this would be to add entries for services on my NAS.

**Comment by ainar-g** [comment]
That sounds like something that is already covered by the [`$dnsrewrite`] functionality.

[`$dnsrewrite`]: https://github.com/AdguardTeam/AdGuardHome/wiki/Hosts-Blocklists#dnsrewrite
